nemo/collections/tts/models/fastpitch.py

```diff
@@ -137,9 +155,22 @@ def parser(self):
             return self._parser
 
         if self.learn_alignment:
-            vocab = AudioToCharWithDursF0Dataset.make_vocab(**self._cfg.train_ds.dataset.vocab)
-            self._parser = vocab.encode
+            ds_class_name = self._cfg.train_ds.dataset.split(".")[-1]
+
+            if ds_class_name == "AudioToCharWithPriorAndPitchDataset":
+                logging.warning(
+                    "AudioToCharWithPriorAndPitchDataset will be deprecated. "
+                    "Please change your model to use Torch TTS Collection instead (e.g. see nemo.collections.tts.torch.data.TTSDataset)."
+                )
```

Comment:
Let's plan deprecation for 1.8.
Let's deprecated `fastpitch.yaml`, `fastpitch_align.yaml`, `fastpitch_align_44100.yaml`, and just have your yaml be the default fastpitch yaml for 1.8.
Let's call your yaml: `fastpitch_align_v2.yaml`